Spanner/src/StructType.php

```diff
@@ -54,6 +54,8 @@ class StructType
     private $fields = [];
 
     /**
+     * Class constructor.
+     *
```

Comment:
The `StructTypeTest::__construct` snippet test fails without this. I spent a lot of time digging but I don't know why.